File: 0485-max-consecutive-ones/0485-max-consecutive-ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int maxi=0,cunt=0;
        int n=nums.size();
        for(int i=0;i<n;i++)
        {
            if(nums[i] == 1)
            {
                cunt++;
                maxi=max(maxi,cunt);
            }
            else
            {
                cunt=0;
            }
        }
        return maxi;
    }

};

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        
        j = 0 
        n = len(nums)
        cc = 0
        maxi = 0
        while j < n:
            if nums[j] == 1:
                cc += 1
            else :
                maxi = max(maxi, cc)
                cc = 0
            j += 1
        return max(maxi,cc)